### PR TITLE
Adding FLINT (+NTL), fmt, GEOS, LERC, libaio, libcerf, libcroco, libtirpc, libdap (+PCRE) for GCC(core) 13.3.0

### PR DESCRIPTION
**Pull request by ankekreuzer**
These packages are used in the new SW stage at JSC and were more or less version bumps from earlier toolchains.

### CODE REVIEW

**Comment by ankekreuzer** [comment]
Test report by @ankekreuzer
**SUCCESS**
Build succeeded for 11 out of 11 (11 easyconfigs in total)
zam329 - Linux Rocky Linux 9.4 (Blue Onyx), x86_64, Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz, Python 3.9.18
See https://gist.github.com/ankekreuzer/284b9380b1ae2bc6f91233b9364e2f67 for a full test report.

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen3

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/g/GEOS/GEOS-3.12.2-GCC-13.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+homepage = 'https://trac.osgeo.org/geos'
+description = """GEOS (Geometry Engine - Open Source) is a C++ port of the Java Topology Suite (JTS)"""
+
+toolchain = {'name': 'GCC', 'version': '13.3.0'}
+
+source_urls = ['https://download.osgeo.org/geos/']
+sources = [SOURCELOWER_TAR_BZ2]
+checksums = ['34c7770bf0090ee88488af98767d08e779f124fa33437e0aabec8abd4609fec6']
+
+builddependencies = [('CMake', '3.29.3')]
```
Comment (boegel) [inline easybuild/easyconfigs/g/GEOS/GEOS-3.12.2-GCC-13.3.0.eb:15]: @ankekreuzer Existing recent `GEOS` easyconfigs also have this, we should retain that?

```python
# Build static and shared libraries (2/2)
configopts = ['', '-DBUILD_SHARED_LIBS=OFF']
```

as well as:

```python
toolchain = ...
toolchainopts = {'pic': True}
```

Did you run into trouble with this somehow?

**Review by boegel — DISMISSED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 12 out of 12 (11 easyconfigs in total)
node3116.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/f0f9f2b212c26b3bfabbba97e48f22c3 for a full test report.

File: easybuild/easyconfigs/g/GEOS/GEOS-3.12.2-GCC-13.3.0.eb (hunk 1 above)
Comment (ankekreuzer) [reply easybuild/easyconfigs/g/GEOS/GEOS-3.12.2-GCC-13.3.0.eb:15]: No, it looks like our version at JSC didn't had this. I'll add it here and also in our repo at JSC (we try keep as close as possible to "official" repo). Thanks for spotting this!

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/g/GEOS/GEOS-3.12.2-GCC-13.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+# Build static and shared libraries
+configopts = ['', '-DBUILD_SHARED_LIBS=OFF']
+
+sanity_check_paths = {
+    'files': ['bin/geos-config', 'lib/libgeos.%s' % SHLIB_EXT, 'lib/libgeos_c.so', 'include/geos.h'],
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/g/GEOS/GEOS-3.12.2-GCC-13.3.0.eb:22]: I would suggest to also keep the check for `lib/libgeos.a` to check if the static build was successful:
```suggestion
    'files': ['bin/geos-config', 'lib/libgeos.%s' % SHLIB_EXT, 'lib/libgeos.a', 'lib/libgeos_c.%s' % SHLIB_EXT,
              'include/geos.h'],
```

**Review by SebastianAchilles — APPROVED** [review]
lgtm

**Comment by SebastianAchilles** [comment]
Going in, thanks @ankekreuzer!

**Pull request merged**